CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This release adds `Clone` and `Copy` implementations.
 
 ### [Added]
 
+- add `Methods::merge_replace` ([#1058](https://github.com/paritytech/jsonrpsee/pull/1058))
```

Comment:
revert this or add a new `UNRELEASED section`.

we already got a script that generates the CHANGELOG for us so easiest just to remove this and deal with it in a release PR.